spyder/config/main.py

```diff
@@ -269,6 +269,7 @@
               'show_fullpath': False,
               'show_all_files': False,
               'group_cells': True,
+              'show_files_sorted': False,

```

Comment:
What does this `sorted` mean? Alphabetically or by the Editor's order?

Reply:
I think this is a not so good choice of words that is applied to all your work in this PR and (IMHO) it should be changed.

Reply:
Alphabetically. I agree it is confusing. Do you have something in mind?

Reply:
I can rename the option to `show_files_in_alphabetical_order` or `show_files_sorted_alphabetically`, but it is a little bit long.

Reply:
What about `sort_files_alphabetically`?

Reply:
Agreed as well. How about ``show_files_alphabetically``, ``show_files_alphabetized`` or even ``alphabetize_files``?

Reply:
>What about ``sort_files_alphabetically``?

I like that best.

Reply:
I think I like `show_files_alphabetically` better than `sort_files_alphabetically`.

Maybe I am wrong about that, but the former add a nuance that this is not a permanent sorting we are doing and that this operation is only limited to what is shown in the Outline Explorer.

Reply:
Ok, majority wins, I'll use `sort_files_alphabetically` then.

Reply:
Yeah, good point. Its very subtle, but if you think it could be confusing then yeah best be safe.

Reply:
I'll change the label of the action to `Sort files alphabetically` as well.

Reply:
^^

Reply:
Thanks @jnsebgosselin!